test/index.test.js

```diff
@@ -180,7 +180,30 @@ describe('index', () => {
                 .then(() => assert.fail('Should not get here'))
                 .catch((error) => {
                     assert.calledWith(requestMock, expectedOptions);
-                    assert.match(error.message, 'STATUS CODE 404');
+                    assert.match(error.message, 'Cannot find repository');
```

Comment:
Should error.message be `Cannot find repository https://batman@bitbucket.org/batman/test.git#mynewbranch`?